main/electron.ts

```diff
@@ -32,7 +32,13 @@ export function createWindow(): void {
     height: 970,
     show: false,
     backgroundColor: '#30404d',
-    webPreferences: { nodeIntegration: true, webSecurity: true, enableRemoteModule: false },
+    webPreferences: {
+      nodeIntegration: false,
+      webSecurity: true,
+      enableRemoteModule: true,
```

Comment:
It took a lot of work to get the remote module disabled. We don't want it enabled again.